fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/patient/ChunkReader.java

```diff
@@ -192,6 +192,8 @@ public Object readItem() throws Exception {
                 auditLogger.logSearchOnExport(ctx.getPartitionResourceType(), queryParameters, resources.size(), startTime, endTime, Response.Status.OK, "StorageProvider@" + ctx.getSource(), "BulkDataOperator");
             }
 
+            pageNum++;
```

Comment:
why are we moving the page forward again here?

Reply:
We're passed the first page

Reply:
pageNum is incremented on line 148 for each call to readItem.  And I think we should only be reading one page per readItem, so I don't understand.

Reply:
totally right.. not sure where it came from.. Just tested with a small page size of 100.